datadog_checks_dev/datadog_checks/dev/tooling/utils.py

```diff
@@ -62,14 +62,50 @@ def string_to_toml_type(s):
     return s
 
 
+def complete_set_root(args):
+    """Set the root directory within the context of a cli completion operation."""
+
+    # if we have already set to something other than empty string break
+    if get_root() != '':
+        return
+
+    root = os.getenv('DDEV_ROOT', '')
+    if root and os.path.isdir(root):
+        set_root(root)
+    else:
+        config = load_config()
+        repo_map = {
+            '--core': 'core',
+            '-c': 'core',
+            '--extras': 'extras',
+            '-e': 'extras',
+            '--agent': 'agent',
+            '-a': 'agent',
+            '--here': 'here',
+            '-x': 'here',
+        }
```

Comment:
Is this map already defined somewhere else maybe?

Reply:
Not exactly, would be a good candidate for adding to `constants.py` though..